README.md

```diff
@@ -772,7 +772,7 @@ of record ids, this can be used to help model hierarchical relationships between
 ```typescript
 const pc = new Pinecone();
 const index = pc.index('my-index').namespace('my-namespace');
-const results = await index.list({ prefix: 'doc1' });
+const results = await index.listPaginated({ prefix: 'doc1' });
```

Comment:
I've been told that passing a prefix without some special separator can result in some unexpected results in some cases, so we should use a separator, I think:
```suggestion
const results = await index.listPaginated({ prefix: 'doc1#' });
```

Reply:
Good callout, thanks!